src/plays/savings-calculator/readme.md

```diff
@@ -0,0 +1,27 @@
+# Savings Calculator 💰
+
+A simple savings calculator where one can enter starting balance, monthly contribution, period/tenure and annual interest rate to get the end balance after the period/tenure.
+
+## Play Demographic
+
+- Language: JavaScript
+- Level: Beginner
+
+## Creator Information
+
+User ID: kd100100
+
+- [GitHub](https://github.com/kd100100)
+- [LinkedIn](https://www.linkedin.com/in/keerthivasand/)
+
+## What will you learn?
+
+The Savings Calculator project demonstrates
+
+- Event Handling
+- State Management
+- SCSS Integration
+- Unit Testing using Jest
+- Responsive Styling
+
+# Happy Learning! 🤩
```

Comment:
@kd100100 , Can you add some implementation details?
You can refer to other plays.

Reply:
Added implementation details